winui/Avatar-view/customization.md

```diff
@@ -1,10 +1,149 @@
 ---
 layout: post
 title: Customization in WinUI AvatarView control | Syncfusion
-description: Learn all about the customization features such as Height, Width, BorderBrush, Background, FontSize and CornerRadius in WinUI AvatarView control here.
+description: Learn all about the customization features such as BorderThickness, BorderBrush, Background, FontSize and GradientBackground in WinUI AvatarView control here.
 platform: WinUI
 control: AvatarView
 documentation: ug
 ---
 
-# Customization in WinUI AvatarView 
\ No newline at end of file
+# Customization in WinUI AvatarView 
+
+The `AvatarView` control offers a range of customization options, including the ability to modify its colors, background, and more. You can utilize the following properties to customize this control:
+
+## Border
+
+Personalize the border of `AvatarView` control by adjusting the border brush and border thickness properties to achieve the desired look.
+
+### BorderBrush
+
+To set the color of the border surrounding your `AvatarView`, you can use the `BorderBrush` property.
+
+{% tabs %}
+{% highlight xaml %}
+
+<syncfusion:SfAvatarView  ContentType="AvatarCharacter" 
+                          AvatarCharacter="Avatar16"
+                          AvatarSize="ExtraLarge"
+                          BorderBrush="Red">
+</syncfusion:SfAvatarView>
+
+{% endhighlight %}
+{% highlight c# %}
+
+SfAvatarView avatarView = new SfAvatarView();
+avatarView.ContentType = AvatarContentType.AvatarCharacter;
+avatarView.AvatarShape = AvatarShape.Circle;
+avatarView.AvatarSize = AvatarSize.ExtraLarge;
+avatarView.AvatarCharacter = AvatarCharacter.Avatar16;
+avatarView.BorderBrush = new SolidColorBrush(Colors.Red);
+
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AvatarView control with custom BorderBrush](avatarview_images/winui_borderbrush_avatarview.png)
+
+### Border Thickness
+
+You can customize the thickness of the AvatarView control by using the `BorderThickness` property.
+
+{% tabs %}
+{% highlight xaml %}
+
+<syncfusion:SfAvatarView   AvatarShape="Custom"
+                           Width="100"
+                           Height="100"
+                           CornerRadius="50"
+                           BorderBrush="Black" 
+                           BorderThickness="4">
+</syncfusion:SfAvatarView>
+
+{% endhighlight %}
+{% highlight c# %}
+
+SfAvatarView avatarView = new SfAvatarView();
+avatarView.AvatarShape = AvatarShape.Custom;
+avatarView.Height = 100;
+avatarView.Width = 100;
+avatarView.BorderThickness = new Thickness(4);
+avatarView.CornerRadius = new CornerRadius(50);
+avatarView.BorderBrush = new SolidColorBrush(Colors.Black);
+
+{% endhighlight %}
+{% endtabs %}
+
+![WinUI AvatarView control with custom BorderThickness](avatarview_images/winui_borderthickness_avatarview.png)
+
+### Background
+
```

Comment:
@MadhumithanJayapathi ## Background

Reply:
@Manivannan-E Changed the ###Background to ##Background.